api/edge_api/identities/serializers.py

```diff
@@ -114,12 +114,12 @@ def save(self, **kwargs):
                 raise serializers.ValidationError(
                     "Feature state already exists."
                 ) from e
-        if self.validated_data.get("multivariate_feature_state_values"):
-            self.instance.multivariate_feature_state_values = self.validated_data[
-                "multivariate_feature_state_values"
-            ]
-
         self.instance.set_value(feature_state_value)
+        self.instance.enabled = self.validated_data["enabled"]
```

Comment:
Do we allow PATCH requests to this endpoint or only PUT? Seems like we might need to disable PATCH for this code to work.

Reply:
I am pretty sure we don't do PATCH, but let me ask Kyle before disabling it

Reply:
Yeah, we don't use PATCH